sass/components/_categories.scss

```diff
@@ -0,0 +1,140 @@
+@charset 'UTF-8';
+
+.modal {
+    .edit-category {
+        &.modal-dialog {
+            margin-top: calc(50vh - 240px);
+        }
+
+        .modal-body {
+            max-height: 100%;
+            padding: 0;
+
+            .form-control {
+                border: 2px solid var(--button-bg);
+                box-sizing: border-box;
+                border-radius: 4px;
+                height: 40px;
+            }
+
+            .input-clear {
+                font-style: normal;
+                font-weight: normal;
+                font-size: 28px;
+                line-height: 16px;
+                color: var(--center-channel-color);
+                opacity: 0.48;
+                right: 36px;
+                top: 10px;
+                width: 16px;
+                height: 16px;
+            }
+        }
+
+        .modal-content {
+            border-radius: 8px;
+        }
+
+        .modal-header {
+            background: transparent;
+            border: none;
+            min-height: 0;
+            padding: 0;
+
+            .close {
+                color: v(center-channel-color-56);
```

Comment:
Is`v()` an scss mixin/function for `--var()`?

Reply:
Not sure, this was copy/pasted from the channel switcher modal.
But I would wager yes.

Reply:
Given the potential confusion, can we please change this for the proper --var()? It's only an extra 4 characters and then removes the need for a mixin/function. I realize it's used elsewhere and I will suggest the same in those cases if we refactor/rebuild.